pkg/resource/deploy/source_eval.go

```diff
@@ -156,14 +157,23 @@ func (src *evalSource) Iterate(ctx context.Context, providers ProviderSource) (S
 		return nil, fmt.Errorf("failed to start resource monitor: %w", err)
 	}
 
+	// Also start up a schema loader for the language runtime to use to fetch schema information.
+	loader := schema.NewPluginLoader(src.plugctx.Host)
+	loaderServer := schema.NewLoaderServer(loader)
+	grpcServer, err := plugin.NewServer(src.plugctx, schema.LoaderRegistration(loaderServer))
```

Comment:
Nit: maybe inline the line above so that this can be `loaderServer`? For some stupid reason I'm what-iffing we have more than one server in this code block later on, but feel free to ignore me.

Reply:
Yeh good call

Reply:
Although technically everything should be able to initialise under one server here, but that requires some fixups through the other bits of code that make the engine and monitor servers as well.